app/Http/Controllers/Assets/AssetCheckinController.php

```diff
@@ -75,6 +76,10 @@ public function store(AssetCheckinRequest $request, $assetId = null, $backto = n
 
         $this->authorize('checkin', $asset);
 
+        if ($request->filled('log_audit') == "1") {
+            $this->authorize('audit', Asset::class);
```

Comment:
This will cause a forbidden error. We don't need the entire block in this section.